modules/social_features/social_user/social_user.module

```diff
@@ -200,15 +200,18 @@ function social_user_form_user_register_form_alter(&$form, FormStateInterface $f
       ];
     }
 
-    $login_link = Link::createFromRoute(new TranslatableMarkup('Log in'), 'user.login', [], $link_options)->toString();
-
-    $form['account']['login-link'] = [
-      '#markup' => new TranslatableMarkup("Have an account already? @link", ["@link" => $login_link]),
-      '#weight' => 1000,
-      '#cache' => [
-        'contexts' => ['url.query_args'],
-      ],
-    ];
+    // Only show for anonymous.
+    if (true === \Drupal::currentUser()->isAnonymous()) {
```

Comment:
```suggestion
    if (TRUE === \Drupal::currentUser()->isAnonymous()) {
```

Reply:
Drupal vs. the rest of the world 👎

Reply:
Haha, TRUE story ;p

Reply:
I added a suggestion to eliminate the point of contention.